### PR TITLE
Fixed tasks: Step 5 human performance, Step 2 Verifying Multi-sentence-ness, wikiHow Step Membership, Commonsense Misinformation Tracking Pilot [cancer data setup] 10, and more and deleting TAP tests duplicates

### PR DESCRIPTION
**Pull request by klxu03**
Also skipping more tests where inputs/questions are hidden behind another click

Waiting for TAP tests to confirm Step 2, changed order not sure if enough to fix

@danyaljj

### CODE REVIEW

File: src/evaluation/baselines.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def solve(self, input: Input, **kwargs):
                 r1 = self.actions.wait_for_element(input)
 
                 # wait 0.1 sec for the page to fully load
-                sleep(0.1)
+                sleep(3)
```
Comment (danyaljj) [inline src/evaluation/baselines.py]: This may make things a lot slower though

Comment (danyaljj) [reply src/evaluation/baselines.py]: Nvm saw your slack message.

Comment (klxu03) [reply src/evaluation/baselines.py]: No you're 100% right! This shouldn't have made it into prod, this is only for run_single when analyzing a single instance. Fixing right now

**Review by danyaljj — APPROVED** [review]
LSGM! 💯  Merge when you're happy!
